
PLAY [reco_release] **************************************************************************************************************

TASK [Gathering Facts] ***********************************************************************************************************
ok: [10.11.5.114]
ok: [10.11.5.113]
ok: [10.11.5.112]
ok: [10.11.5.111]
ok: [10.11.5.115]
ok: [10.11.5.117]
ok: [10.11.5.116]
ok: [10.11.5.119]
ok: [10.11.5.120]
ok: [10.11.5.118]
ok: [10.11.5.121]
ok: [10.11.5.123]
ok: [10.11.5.124]
ok: [10.11.5.125]
ok: [10.11.5.122]
ok: [10.11.5.126]
ok: [10.11.5.128]
ok: [10.11.5.127]
ok: [10.11.5.129]
ok: [10.11.5.130]
ok: [10.11.5.131]
ok: [10.11.5.132]
ok: [10.11.5.134]
ok: [10.11.5.133]

TASK [reco_release : pull reco images 123] ***************************************************************************************
fatal: [10.11.5.112]: FAILED! => {"changed": true, "cmd": "docker pull 123", "delta": "0:00:03.863461", "end": "2018-10-26 15:26:54.271636", "msg": "non-zero return code", "rc": 1, "start": "2018-10-26 15:26:50.408175", "stderr": "Error response from daemon: pull access denied for 123, repository does not exist or may require 'docker login'", "stderr_lines": ["Error response from daemon: pull access denied for 123, repository does not exist or may require 'docker login'"], "stdout": "Using default tag: latest", "stdout_lines": ["Using default tag: latest"]}
fatal: [10.11.5.111]: FAILED! => {"changed": true, "cmd": "docker pull 123", "delta": "0:00:03.936891", "end": "2018-10-26 15:26:54.326845", "msg": "non-zero return code", "rc": 1, "start": "2018-10-26 15:26:50.389954", "stderr": "Error response from daemon: pull access denied for 123, repository does not exist or may require 'docker login'", "stderr_lines": ["Error response from daemon: pull access denied for 123, repository does not exist or may require 'docker login'"], "stdout": "Using default tag: latest", "stdout_lines": ["Using default tag: latest"]}
fatal: [10.11.5.113]: FAILED! => {"changed": true, "cmd": "docker pull 123", "delta": "0:00:03.928265", "end": "2018-10-26 15:26:54.370945", "msg": "non-zero return code", "rc": 1, "start": "2018-10-26 15:26:50.442680", "stderr": "Error response from daemon: pull access denied for 123, repository does not exist or may require 'docker login'", "stderr_lines": ["Error response from daemon: pull access denied for 123, repository does not exist or may require 'docker login'"], "stdout": "Using default tag: latest", "stdout_lines": ["Using default tag: latest"]}
fatal: [10.11.5.115]: FAILED! => {"changed": true, "cmd": "docker pull 123", "delta": "0:00:03.860316", "end": "2018-10-26 15:26:54.462355", "msg": "non-zero return code", "rc": 1, "start": "2018-10-26 15:26:50.602039", "stderr": "Error response from daemon: pull access denied for 123, repository does not exist or may require 'docker login'", "stderr_lines": ["Error response from daemon: pull access denied for 123, repository does not exist or may require 'docker login'"], "stdout": "Using default tag: latest", "stdout_lines": ["Using default tag: latest"]}
fatal: [10.11.5.114]: FAILED! => {"changed": true, "cmd": "docker pull 123", "delta": "0:00:04.119248", "end": "2018-10-26 15:26:54.669408", "msg": "non-zero return code", "rc": 1, "start": "2018-10-26 15:26:50.550160", "stderr": "Error response from daemon: pull access denied for 123, repository does not exist or may require 'docker login'", "stderr_lines": ["Error response from daemon: pull access denied for 123, repository does not exist or may require 'docker login'"], "stdout": "Using default tag: latest", "stdout_lines": ["Using default tag: latest"]}
fatal: [10.11.5.118]: FAILED! => {"changed": true, "cmd": "docker pull 123", "delta": "0:00:03.959894", "end": "2018-10-26 15:26:58.787586", "msg": "non-zero return code", "rc": 1, "start": "2018-10-26 15:26:54.827692", "stderr": "Error response from daemon: pull access denied for 123, repository does not exist or may require 'docker login'", "stderr_lines": ["Error response from daemon: pull access denied for 123, repository does not exist or may require 'docker login'"], "stdout": "Using default tag: latest", "stdout_lines": ["Using default tag: latest"]}
fatal: [10.11.5.120]: FAILED! => {"changed": true, "cmd": "docker pull 123", "delta": "0:00:03.867343", "end": "2018-10-26 15:26:58.918549", "msg": "non-zero return code", "rc": 1, "start": "2018-10-26 15:26:55.051206", "stderr": "Error response from daemon: pull access denied for 123, repository does not exist or may require 'docker login'", "stderr_lines": ["Error response from daemon: pull access denied for 123, repository does not exist or may require 'docker login'"], "stdout": "Using default tag: latest", "stdout_lines": ["Using default tag: latest"]}
fatal: [10.11.5.117]: FAILED! => {"changed": true, "cmd": "docker pull 123", "delta": "0:00:04.202851", "end": "2018-10-26 15:26:59.007806", "msg": "non-zero return code", "rc": 1, "start": "2018-10-26 15:26:54.804955", "stderr": "Error response from daemon: pull access denied for 123, repository does not exist or may require 'docker login'", "stderr_lines": ["Error response from daemon: pull access denied for 123, repository does not exist or may require 'docker login'"], "stdout": "Using default tag: latest", "stdout_lines": ["Using default tag: latest"]}
fatal: [10.11.5.119]: FAILED! => {"changed": true, "cmd": "docker pull 123", "delta": "0:00:04.293195", "end": "2018-10-26 15:26:59.198981", "msg": "non-zero return code", "rc": 1, "start": "2018-10-26 15:26:54.905786", "stderr": "Error response from daemon: pull access denied for 123, repository does not exist or may require 'docker login'", "stderr_lines": ["Error response from daemon: pull access denied for 123, repository does not exist or may require 'docker login'"], "stdout": "Using default tag: latest", "stdout_lines": ["Using default tag: latest"]}
fatal: [10.11.5.116]: FAILED! => {"changed": true, "cmd": "docker pull 123", "delta": "0:00:04.837529", "end": "2018-10-26 15:26:59.549672", "msg": "non-zero return code", "rc": 1, "start": "2018-10-26 15:26:54.712143", "stderr": "Error response from daemon: pull access denied for 123, repository does not exist or may require 'docker login'", "stderr_lines": ["Error response from daemon: pull access denied for 123, repository does not exist or may require 'docker login'"], "stdout": "Using default tag: latest", "stdout_lines": ["Using default tag: latest"]}
fatal: [10.11.5.121]: FAILED! => {"changed": true, "cmd": "docker pull 123", "delta": "0:00:03.938644", "end": "2018-10-26 15:27:03.117935", "msg": "non-zero return code", "rc": 1, "start": "2018-10-26 15:26:59.179291", "stderr": "Error response from daemon: pull access denied for 123, repository does not exist or may require 'docker login'", "stderr_lines": ["Error response from daemon: pull access denied for 123, repository does not exist or may require 'docker login'"], "stdout": "Using default tag: latest", "stdout_lines": ["Using default tag: latest"]}
fatal: [10.11.5.123]: FAILED! => {"changed": true, "cmd": "docker pull 123", "delta": "0:00:03.937425", "end": "2018-10-26 15:27:03.258846", "msg": "non-zero return code", "rc": 1, "start": "2018-10-26 15:26:59.321421", "stderr": "Error response from daemon: pull access denied for 123, repository does not exist or may require 'docker login'", "stderr_lines": ["Error response from daemon: pull access denied for 123, repository does not exist or may require 'docker login'"], "stdout": "Using default tag: latest", "stdout_lines": ["Using default tag: latest"]}
fatal: [10.11.5.124]: FAILED! => {"changed": true, "cmd": "docker pull 123", "delta": "0:00:03.901110", "end": "2018-10-26 15:27:03.485571", "msg": "non-zero return code", "rc": 1, "start": "2018-10-26 15:26:59.584461", "stderr": "Error response from daemon: pull access denied for 123, repository does not exist or may require 'docker login'", "stderr_lines": ["Error response from daemon: pull access denied for 123, repository does not exist or may require 'docker login'"], "stdout": "Using default tag: latest", "stdout_lines": ["Using default tag: latest"]}
fatal: [10.11.5.125]: FAILED! => {"changed": true, "cmd": "docker pull 123", "delta": "0:00:03.809913", "end": "2018-10-26 15:27:03.666708", "msg": "non-zero return code", "rc": 1, "start": "2018-10-26 15:26:59.856795", "stderr": "Error response from daemon: pull access denied for 123, repository does not exist or may require 'docker login'", "stderr_lines": ["Error response from daemon: pull access denied for 123, repository does not exist or may require 'docker login'"], "stdout": "Using default tag: latest", "stdout_lines": ["Using default tag: latest"]}
fatal: [10.11.5.122]: FAILED! => {"changed": true, "cmd": "docker pull 123", "delta": "0:00:05.900743", "end": "2018-10-26 15:27:05.157428", "msg": "non-zero return code", "rc": 1, "start": "2018-10-26 15:26:59.256685", "stderr": "Error response from daemon: pull access denied for 123, repository does not exist or may require 'docker login'", "stderr_lines": ["Error response from daemon: pull access denied for 123, repository does not exist or may require 'docker login'"], "stdout": "Using default tag: latest", "stdout_lines": ["Using default tag: latest"]}
fatal: [10.11.5.126]: FAILED! => {"changed": true, "cmd": "docker pull 123", "delta": "0:00:03.973093", "end": "2018-10-26 15:27:07.468388", "msg": "non-zero return code", "rc": 1, "start": "2018-10-26 15:27:03.495295", "stderr": "Error response from daemon: pull access denied for 123, repository does not exist or may require 'docker login'", "stderr_lines": ["Error response from daemon: pull access denied for 123, repository does not exist or may require 'docker login'"], "stdout": "Using default tag: latest", "stdout_lines": ["Using default tag: latest"]}
fatal: [10.11.5.128]: FAILED! => {"changed": true, "cmd": "docker pull 123", "delta": "0:00:03.838074", "end": "2018-10-26 15:27:07.667045", "msg": "non-zero return code", "rc": 1, "start": "2018-10-26 15:27:03.828971", "stderr": "Error response from daemon: pull access denied for 123, repository does not exist or may require 'docker login'", "stderr_lines": ["Error response from daemon: pull access denied for 123, repository does not exist or may require 'docker login'"], "stdout": "Using default tag: latest", "stdout_lines": ["Using default tag: latest"]}
fatal: [10.11.5.129]: FAILED! => {"changed": true, "cmd": "docker pull 123", "delta": "0:00:04.381342", "end": "2018-10-26 15:27:08.358730", "msg": "non-zero return code", "rc": 1, "start": "2018-10-26 15:27:03.977388", "stderr": "Error response from daemon: pull access denied for 123, repository does not exist or may require 'docker login'", "stderr_lines": ["Error response from daemon: pull access denied for 123, repository does not exist or may require 'docker login'"], "stdout": "Using default tag: latest", "stdout_lines": ["Using default tag: latest"]}
fatal: [10.11.5.130]: FAILED! => {"changed": true, "cmd": "docker pull 123", "delta": "0:00:03.967413", "end": "2018-10-26 15:27:09.495177", "msg": "non-zero return code", "rc": 1, "start": "2018-10-26 15:27:05.527764", "stderr": "Error response from daemon: pull access denied for 123, repository does not exist or may require 'docker login'", "stderr_lines": ["Error response from daemon: pull access denied for 123, repository does not exist or may require 'docker login'"], "stdout": "Using default tag: latest", "stdout_lines": ["Using default tag: latest"]}
fatal: [10.11.5.131]: FAILED! => {"changed": true, "cmd": "docker pull 123", "delta": "0:00:03.946490", "end": "2018-10-26 15:27:11.769996", "msg": "non-zero return code", "rc": 1, "start": "2018-10-26 15:27:07.823506", "stderr": "Error response from daemon: pull access denied for 123, repository does not exist or may require 'docker login'", "stderr_lines": ["Error response from daemon: pull access denied for 123, repository does not exist or may require 'docker login'"], "stdout": "Using default tag: latest", "stdout_lines": ["Using default tag: latest"]}
fatal: [10.11.5.132]: FAILED! => {"changed": true, "cmd": "docker pull 123", "delta": "0:00:04.029625", "end": "2018-10-26 15:27:11.944948", "msg": "non-zero return code", "rc": 1, "start": "2018-10-26 15:27:07.915323", "stderr": "Error response from daemon: pull access denied for 123, repository does not exist or may require 'docker login'", "stderr_lines": ["Error response from daemon: pull access denied for 123, repository does not exist or may require 'docker login'"], "stdout": "Using default tag: latest", "stdout_lines": ["Using default tag: latest"]}
fatal: [10.11.5.133]: FAILED! => {"changed": true, "cmd": "docker pull 123", "delta": "0:00:03.904206", "end": "2018-10-26 15:27:12.571922", "msg": "non-zero return code", "rc": 1, "start": "2018-10-26 15:27:08.667716", "stderr": "Error response from daemon: pull access denied for 123, repository does not exist or may require 'docker login'", "stderr_lines": ["Error response from daemon: pull access denied for 123, repository does not exist or may require 'docker login'"], "stdout": "Using default tag: latest", "stdout_lines": ["Using default tag: latest"]}
fatal: [10.11.5.127]: FAILED! => {"changed": true, "cmd": "docker pull 123", "delta": "0:00:10.475977", "end": "2018-10-26 15:27:14.065630", "msg": "non-zero return code", "rc": 1, "start": "2018-10-26 15:27:03.589653", "stderr": "Error response from daemon: Get https://registry-1.docker.io/v2/: net/http: TLS handshake timeout", "stderr_lines": ["Error response from daemon: Get https://registry-1.docker.io/v2/: net/http: TLS handshake timeout"], "stdout": "Using default tag: latest", "stdout_lines": ["Using default tag: latest"]}
fatal: [10.11.5.134]: FAILED! => {"changed": true, "cmd": "docker pull 123", "delta": "0:00:04.587722", "end": "2018-10-26 15:27:14.326840", "msg": "non-zero return code", "rc": 1, "start": "2018-10-26 15:27:09.739118", "stderr": "Error response from daemon: pull access denied for 123, repository does not exist or may require 'docker login'", "stderr_lines": ["Error response from daemon: pull access denied for 123, repository does not exist or may require 'docker login'"], "stdout": "Using default tag: latest", "stdout_lines": ["Using default tag: latest"]}
	to retry, use: --limit @/home/kddev/cheng/autohdmap_deploy/reco_release.retry

PLAY RECAP ***********************************************************************************************************************
10.11.5.111                : ok=1    changed=0    unreachable=0    failed=1   
10.11.5.112                : ok=1    changed=0    unreachable=0    failed=1   
10.11.5.113                : ok=1    changed=0    unreachable=0    failed=1   
10.11.5.114                : ok=1    changed=0    unreachable=0    failed=1   
10.11.5.115                : ok=1    changed=0    unreachable=0    failed=1   
10.11.5.116                : ok=1    changed=0    unreachable=0    failed=1   
10.11.5.117                : ok=1    changed=0    unreachable=0    failed=1   
10.11.5.118                : ok=1    changed=0    unreachable=0    failed=1   
10.11.5.119                : ok=1    changed=0    unreachable=0    failed=1   
10.11.5.120                : ok=1    changed=0    unreachable=0    failed=1   
10.11.5.121                : ok=1    changed=0    unreachable=0    failed=1   
10.11.5.122                : ok=1    changed=0    unreachable=0    failed=1   
10.11.5.123                : ok=1    changed=0    unreachable=0    failed=1   
10.11.5.124                : ok=1    changed=0    unreachable=0    failed=1   
10.11.5.125                : ok=1    changed=0    unreachable=0    failed=1   
10.11.5.126                : ok=1    changed=0    unreachable=0    failed=1   
10.11.5.127                : ok=1    changed=0    unreachable=0    failed=1   
10.11.5.128                : ok=1    changed=0    unreachable=0    failed=1   
10.11.5.129                : ok=1    changed=0    unreachable=0    failed=1   
10.11.5.130                : ok=1    changed=0    unreachable=0    failed=1   
10.11.5.131                : ok=1    changed=0    unreachable=0    failed=1   
10.11.5.132                : ok=1    changed=0    unreachable=0    failed=1   
10.11.5.133                : ok=1    changed=0    unreachable=0    failed=1   
10.11.5.134                : ok=1    changed=0    unreachable=0    failed=1   

[Finish] Failed to deploy [123]